### ISSUE TITLE
Дополнить документацию

### ISSUE THREAD

**Issue by AlexeyKorshun** [hhru/android-multimodule-plugin#40]
Не нашел примеров что делают доступные выражения, можно рядом с ними привести пример что получится.

**Comment by PStrelchenko** [comment]
Привет!
Ок, это действительно стоит сделать.

Пока документацию не доработали, можно посмотреть на тесты -- https://github.com/hhru/android-multimodule-plugin/tree/master/shared/feature/geminio-sdk/src/test/kotlin/ru/hh/plugins/geminio/sdk/recipe/expressions/modifiers

Из них должно быть понятно, что делает каждый модификатор.

**Comment by AlexeyKorshun** [comment]
да, я это видел, но не все знают, что тесты есть, и лезть туда не всегда хотят, 
Еще появилось желание дополнить выражением uppercase() и lowercase()
